redaxo/src/core/lib/console/package/package.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @package redaxo\core
+ */
+abstract class rex_command_package extends rex_console_command
+{
+    protected function configure()
+    {
+        $this->addArgument('package-id', InputArgument::REQUIRED, 'The id of the package (addon or plugin); e.g. "cronjob" or "structure/content"');
+        $this->configureCommand();
```

Comment:
Falls es bei der Abstraktion bleibt, würde ich für `configure` nicht diese neue Methode einführen, sondern die Original-Methode in den Kindklassen erneuten aufrufen, und da dann die Parant-Methoden aufrufen.